lib/container_orchestrator/object_definition.rb

```diff
@@ -107,6 +107,7 @@ def secret_definition(name, string_data)
     def default_environment
       [
         {:name => "GUID",                    :value => MiqServer.my_guid},
+        {:name => "HOME",                    :value => Rails.root.join("tmp").to_s},
```

Comment:
Note, I believe `rake tmp:create` creates this directory, which is probably called from `tmp:clear` which we call in [bin/setup](https://github.com/ManageIQ/manageiq/blob/123399afd7a4c5dbb66f268816460ec431652476/bin/setup#L40), I don't know if I need to create this location earlier.  I need to test a build with this.

Reply:
I'm not sure Rails tmp makes sense, because it's, well, temporary.

If we do pick a temp dir location, should we have a subdir under it?

Reply:
See https://github.com/ManageIQ/manageiq/pull/21606#issuecomment-990131902.  If we pick a different location, we'll need to make sure it's created beforehand, maybe in the [container_env](https://github.com/ManageIQ/manageiq-pods/blob/master/images/manageiq-base/container-assets/container_env) or before we source it in the [entrypoint script](https://github.com/ManageIQ/manageiq-pods/blob/ba2dfaaaa97aaccffdd088d63f6ce6f721ce7dc1/images/manageiq-base-worker/container-assets/entrypoint#L3).  Note, `container_env` is included in non-worker images such as the orchestrator whereas this code change is only for the worker deployments created by the orchestrator.